app/controllers/api/account.php

```diff
@@ -1345,14 +1348,18 @@
     ->label('sdk.response.code', Response::STATUS_CODE_OK)
     ->label('sdk.response.type', Response::CONTENT_TYPE_JSON)
     ->label('sdk.response.model', Response::MODEL_LOG_LIST)
-    ->param('limit', 25, new Range(0, 100), 'Maximum number of logs to return in response. By default will return maximum 25 results. Maximum of 100 results allowed per request.', true)
-    ->param('offset', 0, new Range(0, APP_LIMIT_COUNT), 'Offset value. The default value is 0. Use this value to manage pagination. [learn more about pagination](https://appwrite.io/docs/pagination)', true)
+    ->param('queries', [], new Queries(new Limit(), new Offset()), 'Array of query strings generated using the Query class provided by the SDK. [Learn more about queries](https://appwrite.io/docs/databases#querying-documents). Only supported methods are limit and offset', true)
```

Comment:
Not sure how const would be used. This is only exactly like this for log endpoints. We can't const the whole param, right? So it would be const for paramName, paramValidator, paramDescription separately. Not sure if we want that, in the end, we have a lot of duplication in other params already too.

Reply:
Oh, you are right. I didn't see the last sentence.